util/db/secrets.go

```diff
@@ -158,5 +158,18 @@ func URIToSecretName(uriType, uri string) (string, error) {
 	h := fnv.New32a()
 	_, _ = h.Write([]byte(uri))
 	host := strings.ToLower(strings.Split(parsedURI.Host, ":")[0])
+	parsedHost := strings.Split(parsedURI.Host, ":")
+
+	//If IPv6 address with port
+	if len(parsedHost) >= 8 {
```

Comment:
Why use `len(parsedHost) >= 8` to determine if it's ipv6 address? There is already a `ParseIP` function in  `net` package to do the work.